package.json

```diff
@@ -1,13 +1,13 @@
 {
   "name": "template-instance",
-  "version": "0.1.0",
+  "version": "0.7.2",
   "description": "A template repo for creating a SourceCred instance",
   "repository": "git@github.com:sourcecred/template-instance.git",
   "author": "SourceCred Team <contact@sourcecred.io>",
   "license": "MIT",
   "private": true,
   "dependencies": {
-    "sourcecred": "0.7.1"
+    "sourcecred": "0.7.2"
```

Comment:
Once 0.7.4 gets released (tonight afaik) you'll be able to actually set a distribution policy that uses the exponential policy (as @elihanover said)

Reply:
Sweet. Will run it hopefully tomorrow. Do you or @elihanover have a suggested discount to try with Maker? Something that perhaps is close'ish to what they have now (75/25 balanced/immediate)? Or perhaps 50/50 balanced/immediate, which is another option they're considering?